semesterly/templates/about.html

```diff
@@ -183,6 +183,18 @@ <h3>JHU '25</h3>
 										class="fab fa-linkedin fa-2x"></i></a>
 							</div>
 						</div>
+						<div class="col-1-3 member">
+							<div class="circular" style="background-image: url(/static/img/jiashu.jpg)"></div>
+							<h4 class="name">Jiashu Yang</h4>
+							<h3>Software Engineer - Trainee</h3>
+							<h3>JHU '25</h3>
+							<div class="social">
+								<a href="/"><i class="fa fa-fw fa-link fa-2x"></i></a>
```

Comment:
Can you get rid of this too? You only want this if you're going to link to a personal website, but right now it's redirecting to `/`